tsfresh/feature_extraction/extraction.py

```diff
@@ -151,8 +153,11 @@ def _extract_features_parallel_per_kind(kind_to_df_map, settings, column_id, col
                                                            settings=settings)
     pool = Pool(settings.n_processes)
 
-    extracted_features = pool.map(partial_extract_features_for_one_time_series, kind_to_df_map.items(),
-                                  chunksize=settings.chunksize)
+    chunksize = _calculate_best_chunksize(kind_to_df_map, settings)
+
+    total_number_of_expected_results = len(kind_to_df_map)
+    extracted_features = tqdm(pool.imap(partial_extract_features_for_one_time_series, kind_to_df_map.items(),
```

Comment:
Would this not be better suited as also being optional, even if default is on, but can be disabled?  If tsfresh runs constantly (in app), all the tqdm outputs are not going to be or any use and are just going to cost additional tiny ticks and fill app logs unnecessarily.  Perhaps a future pull request waiting to happen :)